mmcv/ops/box_iou_rotated.py

```diff
@@ -14,6 +18,46 @@ def box_iou_rotated(bboxes1, bboxes2, mode='iou', aligned=False):
     of bboxes1 and bboxes2, otherwise the ious between each aligned pair of
     bboxes1 and bboxes2.
 
+    coordinates details:
```

Comment:
We may use Notes or some other stand block name in sphinx.
See https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#colored-boxes-note-seealso-todo-and-warnings. 

You may also preview the contents here https://mmcv--1592.org.readthedocs.build/en/1592/api.html#mmcv.ops.nms_rotated